drivers/soundwire/intel_init.c

```diff
@@ -134,6 +136,59 @@ sdw_intel_scan_controller(struct sdw_intel_acpi_info *info)
 	return 0;
 }
 
+#define HDA_DSP_GEN_BASE                0x0
```

Comment:
@bardliao do you really need to add 0x0?

Reply:
> @bardliao do you really need to add 0x0?

Actually not. Will remove it